pkg/csl/download_uk.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package csl
+
+import (
+	"fmt"
+
+	"github.com/moov-io/base/log"
+	"github.com/moov-io/watchman/pkg/download"
+)
+
+// taken from https://www.gov.uk/government/publications/financial-sanctions-consolidated-list-of-targets/consolidated-list-of-targets#contents
+const (
+	ukuri = "https://ofsistorage.blob.core.windows.net/publishlive/2022format/ConList.csv"
```

Comment:
Weird that the UK is hosting a sanctions list on windows.net, so can we offer an env variable to override this?

Reply:
What would you override this with? I think adding an override would be premature as we don't know how that would change and for what reason.

Reply:
Offering an override would allow users of Watchman to cache files in their network rather than relying on the external internet.